packit_service/worker/build/copr_build.py

```diff
@@ -232,17 +234,40 @@ def run_build(
                 "Copr owner not set. Use Copr config file or `--owner` when calling packit CLI."
             )
 
-        self.api.copr_helper.create_copr_project_if_not_exists(
-            project=self.job_project,
-            chroots=list(self.build_targets),
-            owner=owner,
-            description=None,
-            instructions=None,
-            list_on_homepage=self.list_on_homepage,
-            preserve_project=self.preserve_project,
-            additional_repos=self.additional_repos,
-            update_additional_values=(owner == "packit"),
-        )
+        try:
+            self.api.copr_helper.create_copr_project_if_not_exists(
+                project=self.job_project,
+                chroots=list(self.build_targets),
+                owner=owner,
+                description=None,
+                instructions=None,
+                list_on_homepage=self.list_on_homepage,
+                preserve_project=self.preserve_project,
+                additional_repos=self.additional_repos,
+            )
+        except PackitCoprSettingsException as ex:
+            if self.metadata.pr_id:
+
+                table = (
+                    "| field | old value | new value |\n"
+                    "| ----- | --------- | --------- |\n"
+                )
+                for field, (old, new) in ex.fields_to_change.items():
+                    table += f"| {field} | {old} | {new} |\n"
+
+                msg = (
+                    "We need to change settings of your copr project "
+                    "but we are not allowed to do so.\n\n"
+                    "You have two options to fix your build:\n"
+                    "- Give us `admin` rights to your Copr Project. "
+                    "(We have just asked asked for it.)\n"
+                    "- Change the following settings manually\n\n"
+                    f"{table}\n"
+                    "Please, retrigger the build once you fix the issues."
```

Comment:
This message doesn't say:
- which copr repo needs the update;
- why do the settings need to be changed;

How about:

```
Based on the [Packit configuration](permanent link to packit.yaml) 
the settings of the {owner}/{project} Copr project would need to be 
updated as follows:

{table}

Packit was unable to update the settings above as it is missing `admin` 
permissions on the {owner}/{project} Copr project.

To fix this you can do one of the following:

- Grant Packit `admin` permissions on the {owner}/{project} Copr project 
  [on the following page](link to copr repo settings page).
- [Change the above Copr project settings](link to copr repo settings page) 
  manually to match the Packit configuration.
- Update the [Packit configuration](permanent link to packit.yaml) 
  to match the Copr project settings.

Please re-trigger the build, once the issue above is fixed.
```

Reply:
Should we check comments on the PR to avoid duplicate comments? Or is that done in the `pr_comment` call bellow?

Reply:
> which copr repo needs the update;

They should know what they defined..;) But I'll copy your template.

> permanent link to packit.yaml

This will take some time. Can we improve later/in a follow-up? (We don't know the config file name and need search for it once more. And ideally, don't template that URL here but in OGR.)

> Should we check comments on the PR to avoid duplicate comments? Or is that done in the pr_comment call bellow?

- OGR does not care.
- We can, but:
  - More code. (We want to release this ASAP, don't we?)
  - Users will not receive explicit feedback that we really can't work until they fix that. (Text in status is really short.)

Reply:
> More code. (We want to release this ASAP, don't we?)

Same with the copr URLs. Can I send it later as well? (I would like to play with that on stage in the meantime..)

Reply:
I would actually be in favour of dupe comments - we are not able to build their project, so either they fix it promptly or disable builds altogether.

Reply:
The person who has the permissions to do such a fix, might not be around. Others might still want to work on the PR(s) without having it communicated over and over again a thing they already know.

Reply:
@lachmanfrantisek yes, I'm okay to do improvements in follow up PRs.